pkg/kodo/cmd/build_test.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"fmt"
+	"testing"
+
+	cmp "github.com/google/go-cmp/cmp"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	buildv1api "github.com/openshift/api/build/v1"
+	imagev1api "github.com/openshift/api/image/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func getBuildConfig() buildv1api.BuildConfig {
+	return buildv1api.BuildConfig{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "BuildConfig",
+			APIVersion: "build.openshift.io/v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "my-app-docker-build",
+		},
+		Spec: buildv1api.BuildConfigSpec{
+			CommonSpec: buildv1api.CommonSpec{
+				Source: buildv1api.BuildSource{
+					Type: buildv1api.BuildSourceType("Git"),
+					Git: &buildv1api.GitBuildSource{
+						URI: "https://github.com/openshift/ruby-hello-world.git",
+					},
+				},
+				Strategy: buildv1api.BuildStrategy{
+					Type: buildv1api.BuildStrategyType("Docker"),
+				},
+				Output: buildv1api.BuildOutput{
+					To: &corev1.ObjectReference{
+						Kind: "ImageStreamTag",
+						Name: "my-ruby-image:latest",
+					},
+				},
+			},
+		},
+	}
+}
```

Comment:
I don't think a new function is required for this 
I think best practice would be for the test cases to be completely isolated and not have any dependence on external functions

Reply:
I'm not sure i understand what you mean by external function. I just followed the way we wrote test cases in the session with gagan.